.circleci/config.yml

```diff
@@ -64,7 +64,7 @@ jobs:
           path: ./submitted_jobs.txt
           destination: submitted-jobs
 
-  test_staging:
+  test_real:
```

Comment:
I don't follow why you named it `test_real`. Care to explain?

Reply:
The idea here was to abstract the code away from `dev` and `staging` as here we swap the meanings of `test_dev` and `test_staging`

Reply:
agree, not a brilliant one